src/main/java/de/hhu/propra/sharingplatform/model/Item.java

```diff
@@ -45,4 +40,14 @@ private Item() {
     public Item(User owner) {
         this.owner = owner;
     }
+
+    public long getActiveOffers() {
```

Comment:
maybe rename to something which signals better that this function returns the count of active offers and not the active offers?

Reply:
yeah you are right. I will change it next time we touch the class, should be tomorrow or the day after.